src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/MetadataManager.cs

```diff
@@ -465,6 +468,17 @@ public void GetDependenciesDueToMethodCodePresence(ref DependencyList dependenci
             GetDependenciesDueToMethodCodePresenceInternal(ref dependencies, factory, method, methodIL);
         }
 
+        public virtual bool HasConditionalDependenciesDueToMethodGenericDictionary(MethodDesc method)
```

Comment:
This is not called from anywhere. I guess you wanted to call it from `GenericDictionaryNode.HasConditionalStaticDependencies` ? Or remote it?

Reply:
Good catch! I thought I would use it from `MethodGenericDictionaryNode.HasConditionalStaticDependencies` but then realized I don't have a NodeFactory to ask the questions of. Not too much harm from just returning true and then returning no conditional dependencies (it's a couple extra virtual calls that I assume HasConditionalStaticDependencies is trying to avoid).